### ISSUE TITLE
Add generate call for eslintrc config

### ISSUE THREAD

**Issue by rugk** [AtomLinter/linter-eslint#1131]
There is an [`eslint --init` function](https://eslint.org/docs/user-guide/getting-started#global-installation-and-usage), which seems to auto-generate an `.eslintrc` config file. I don't know how that file looks (and whether the CLI asks the user questions or so), but it would be nice top be able to access that in Atom (as a command) too, so one can get started easily.

Of course, however it may not really call eslint for that, because that also seems to install some dependencies or so.

**Comment by skylize** [comment]
This is a nice idea, but also seems a bit like bloat to me. I feel like you should just open your terminal for one-off commands like that.

**Comment by rugk** [comment]
The issue is that you have to have ESLint installed for that (as a console application). And that is often not done when you just install this add-on.

**Comment by skylize** [comment]
This package is not actually intended to run without you installing `eslint` yourself. 

You need a config file to describe what rules to check for. Your config file should have a shared parent directory with the binary being run. And you should absolutely *not* be putting an ESLint config inside your Atom packages folder.

The current behavior of using the binary from the package installation by default is planned for deprecation, due to the numerous issues that arise from people expecting "just works" behavior from the built-in ESLint.

**Comment by rugk** [comment]
> This package is not actually intended to run without you installing eslint yourself.

Hu? But it _does_ work…
Also EsLint is just just JS/NodeJS, all that is available in Atom, so it seems more than reasonable to ship it with Atom.

> people expecting "just works" behavior

But it **does** *just work*. One can install the add-on, add a config file (of course to the project folder, nobody would add it to some Atom config folder…) and it *does* just work.

**Comment by Arcanemagus** [comment]
> But it does work

Somewhat, it only works if you have no additional plugins, anything beyond that and it gets _really_ iffy as to whether it will work or not.

> Also EsLint is just just JS/NodeJS, all that is available in Atom

Yes, which is how all of this is _running_, but shipping a version of ESLint with the package has caused _many_ of the issue here, like:

> nobody would add it to some Atom config folder…

We've had several people do that actually. Many more install ESLint plugins in this package's folder to "make it work", instead of setting up their own instance in their project (or using a global install).

----

tl;dr For some limited instances of "works", you're right that it "just works" right now, but those are _very_ limited, and it's better for people to just set up ESlint correctly in the first place.

Since I don't see a way to accomplish what you are asking using the [ESLint Node.js API](https://eslint.org/docs/developer-guide/nodejs-api) I'm marking this as `wontfix` for now, if at some point they later add that ability there somehow we'd be happy to implement it here!
